easybuild/easyconfigs/n/netCDF/netCDF-4.4.0-intel-2016a.eb

```diff
@@ -0,0 +1,33 @@
+name = 'netCDF'
+version = '4.4.0'
+
+homepage = 'http://www.unidata.ucar.edu/software/netcdf/'
+description = """NetCDF (network Common Data Form) is a set of software libraries 
+ and machine-independent data formats that support the creation, access, and sharing of array-oriented 
+ scientific data."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = [
+    'https://github.com/Unidata/netcdf-c/archive/'
+]
+
+dependencies = [
+    ('HDF5', '1.8.16'),
+    ('cURL', '7.47.0'),
+]
+
+builddependencies = [
+    ('CMake', '3.4.3'),
+    ('Doxygen', '1.8.11'),
+]
+
+# make sure both static and shared libs are built
+configopts = [
+    "-DBUILD_SHARED_LIBS=OFF ",
+    "-DBUILD_SHARED_LIBS=ON ",
+]
```

Comment:
hehe, no, this will make `eb` cycle through the config-build-install steps twice, once with `OFF`, once with `ON`